mindsdb/integrations/handlers/confluence_handler/confluence_api_client.py

```diff
@@ -0,0 +1,118 @@
+from typing import List
+
+import requests
+
+
+class ConfluenceAPIClient:
+    def __init__(self, url: str, username: str, password: str):
+        self.url = url
+        self.username = username
+        self.password = password
+        self.session = requests.Session()
+        self.session.auth = (self.username, self.password)
+        self.session.headers.update({"Accept": "application/json"})
+
+    def get_spaces(
+        self,
+        ids: List[int] = None,
+        keys: List[str] = None,
+        type: str = None,
```

Comment:
`type` may be confused with ta built-in names?

Reply:
Updated to `space_type`.